app/selenium_ui/jsm/pages/agent_pages.py

```diff
@@ -79,10 +79,18 @@ def add_request_comment(self, rte_status):
 
         if rte_status:
             self.wait_until_available_to_switch(ViewCustomerRequestLocators.comment_text_field_RTE)
-            self.get_element(ViewCustomerRequestLocators.comment_tinymce_field).send_keys(comment_text)
+            if self.driver.app_settings.secure:
+                self.get_element(ViewCustomerRequestLocators.comment_tinymce_field).send_keys(comment_text)
+            else:
+                self.action_chains().move_to_element(self.get_element(ViewCustomerRequestLocators.comment_tinymce_field)
+                                                     ).send_keys(comment_text).perform()
             self.return_to_parent_frame()
         else:
-            self.get_element(ViewCustomerRequestLocators.comment_text_field).send_keys(comment_text)
+            if self.driver.app_settings.secure:
+                self.get_element(ViewCustomerRequestLocators.comment_tinymce_field).send_keys(comment_text)
```

Comment:
Looks like a bug here. Did you check this code with rte disabled?

Reply:
Yes, you are right. Fixed it